exercises/concept/annalyns-infiltration/.docs/instructions.md

```diff
@@ -9,7 +9,12 @@ Having found the kidnappers, Annalyn considers which of the following actions sh
 - Fast attack: a fast attack can be made if the knight is sleeping, as it takes time for him to get his armor on, so he will be vulnerable.
 - Spy: the group can be spied upon if at least one of them is awake. Otherwise, spying is a waste of time.
 - Signal prisoner: the prisoner can be signaled using bird sounds if the prisoner is awake and the archer is sleeping, as archers are trained in bird signaling so they could intercept the message.
-- Free prisoner: if the prisoner is awake and the other two characters are sleeping, a sneaky entry into the camp can free the prisoner. This won't work if the prisoner is sleeping, as the prisoner will be startled by the sudden appearance of her friend and the knight and archer will be awoken. The prisoner can also be freed if the archer is sleeping and Annalyn has her pet dog with her, as the knight will be scared by the dog and will withdraw, and the archer can't equip his bow fast enough to prevent the prisoner from being freed.
+- _Free prisoner_: Annalyn can try sneaking into the camp to free the prisoner.
+  This is a risky thing to do and can only succeed in one of two ways:
+  - If Annalyn has her pet dog with her she can rescue the prisoner if the archer is asleep.
+    The knight is scared of the dog and the archer will not have time to get ready before Annalyn and the prisoner can escape.
+  - If Annalyn does not have her dog then she and the prisoner must be very sneaky!
+    Annalyn can free the prisoner if the prisoner is awake and the knight and archer are both sleeping, but if the prisoner is sleeping they can't be rescued: the prisoner would be startled by Annalyn's sudden appearance and wake up the knight and archer.
```

Comment:
Prisoner has to be awake to be rescued so two unit tests are wrong:

`func CanFreePrisoner(knightIsAwake, archerIsAwake, prisonerIsAwake, petDogIsPresent bool) bool {}`

```
=== RUN   TestCanFreePrisoner
=== RUN   TestCanFreePrisoner/All_characters_are_sleeping._Dog_is_present.
    d:\Adam\exercism\go\annalyns-infiltration\annalyns_infiltration_test.go:274: CanFreePrisoner(false,false,false,true) = false; want true
--- FAIL: TestCanFreePrisoner (0.00s)
    --- FAIL: TestCanFreePrisoner/All_characters_are_sleeping._Dog_is_present. (0.00s)
```
and
```
=== RUN   TestCanFreePrisoner
=== RUN   TestCanFreePrisoner/Knight_is_awake._Archer_is_sleeping._Prisoner_is_sleeping._Dog_is_present.
    d:\Adam\exercism\go\annalyns-infiltration\annalyns_infiltration_test.go:274: CanFreePrisoner(true,false,false,true) = false; want true
--- FAIL: TestCanFreePrisoner (0.00s)
    --- FAIL: TestCanFreePrisoner/Knight_is_awake._Archer_is_sleeping._Prisoner_is_sleeping._Dog_is_present. (0.00s)
```

Both of them should expect FALSE

Reply:
@gawelczyk Please open a new issue for discussion instead, this PR has been merged already.